.github/workflows/selenium.yml

```diff
@@ -0,0 +1,61 @@
+# This workflow will run headful selenium tests.
+
+name: Selenium tests
+
+# Declare default permissions as read only.
+permissions: read-all
+
+on:
+  merge_group:
+  pull_request:
+  push:
+    branches: 'main'
+  workflow_dispatch:
```

Comment:
if you don't use the `verbose` option anywhere, delete it from here

Reply:
removed